### PR TITLE
Update kubevirt pages to use new actions

### PR DESCRIPTION
**Pull request by kevinylee1991**

### CODE REVIEW

File: src/app/plugins/kubevirt/components/InformationalLandingPage.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import CardButton from 'core/elements/button/CardButton'
 import { KubeVirtLogo } from '../logo'
 
 export default function InformationalLandingPage() {
+  // Ask john: these buttons don't have any actions?
```
Comment (kevinylee1991) [inline src/app/plugins/kubevirt/components/InformationalLandingPage.tsx:9]: @jruddell

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            selectFirst={false}
+            clusterId={filterValues.clusterId}
+            setInitialNamespace
+            {...filterProps}
+          />
+        ),
+        onChange: getParamsUpdater('namespace'),
+      } as GridFilterSpec<SelectorModel, Params, 'namespace'>,
+      {
+        columnKey: 'kind', // there is no matching column key, using kind to bypass
```
Comment (kevinylee1991) [inline src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:183]: Is there a defined way to add controlled filters that do not correspond to a particular item property?

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+        getParamsUpdater={getParamsUpdater}
+        batchActions={batchActions}
+        multiSelection={false}
+        {...pick(listTablePrefs, params)}
+      />
+    </>
+  )
+}
+
+// ListContainer needs to support this
```
Comment (kevinylee1991) [inline src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:217]: Do we want ListContainer to support the NoContentMessage component like the LIstTable did?

File: src/app/plugins/kubevirt/components/InformationalLandingPage.tsx (hunk 1 above)
Comment (jruddell) [reply src/app/plugins/kubevirt/components/InformationalLandingPage.tsx:9]: no definition yet for what they should do, supposed to hear on monday

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+              <div key={addr}>{addr}</div>
+            ))}
+          </div>
+        </div>
+      ))}
+    </>
+  )
+}
+const NameCell = ({ value, item: vm }) => {
+  const namespace = vm?.metadata?.namespace
```
Comment (jruddell) [inline src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:76]: I have a generic link component for rendering name cells. Lets use that, look at the pods list page for an example

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+    <SimpleLink
+      src={routes.virtualMachineInstances.detail.path({ clusterId, namespace, name: value })}
+    >
+      {value}
+    </SimpleLink>
+  )
+}
+
+const ClusterLinkCell = ({ value, item }) => {
+  const clusterId = item?.clusterId
```
Comment (jruddell) [inline src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:88]: another use case for the generic link cell

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx (hunk 2 above)
Comment (xantorres) [reply src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:183]: You have to create a global filter for that, not a regular filter.

File: src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx (hunk 3 above)
Comment (xantorres) [reply src/app/plugins/kubevirt/components/virtual-machines/VirtualMachineInstancesListPage.tsx:217]: I guess you can use `emptyContent` for that?

**Review by jruddell — APPROVED** [review]

**Pull request merged**